pkg/reconciler/subscription/subscription_test.go

```diff
@@ -224,7 +223,6 @@ func TestAllCases(t *testing.T) {
 					WithSubscriptionReply(subscriberGVK, replyName),
 					// The first reconciliation will initialize the status conditions.
 					WithInitSubscriptionConditions,
-					WithSubscriptionPhysicalSubscriptionSubscriber(subscriberURI),
```

Comment:
interesting, I suppose this is because you are validating. NICE.

Reply:
Yes